ParProcCo/nxdata_aggregator.py

```diff
@@ -251,13 +251,22 @@ def _write_aggregation_file(self, aggregation_output: Path) -> Path:
 
             f.attrs["default"] = self.nxentry_name
 
-            for i, filepath in enumerate(self.data_files):
-                f[f"entry{i}"] = h5py.ExternalLink(str(filepath), self.nxentry_name)
+            i = 0
```

Comment:
Better to enumerate (so files with missing process groups are obvious).

Reply:
And always create old_processed group, even if empty because of all process groups being missing?

Reply:
Probably not in that case.